plugins/inventory/proxmox.py

```diff
@@ -503,10 +515,13 @@ def _handle_item(self, node, ittype, item):
         node_type_group = self._group('%s_%s' % (node, ittype))
         self.inventory.add_child(self._group('all_' + ittype), name)
         self.inventory.add_child(node_type_group, name)
-        if item['status'] == 'stopped':
-            self.inventory.add_child(self._group('all_stopped'), name)
-        elif item['status'] == 'running':
-            self.inventory.add_child(self._group('all_running'), name)
+
+        item_status = item['status']
+        if item_status == 'running':
+            if want_facts and ittype == 'qemu' and self.get_option('qemu_extended_statuses'):
+                # get more details about the status of the qemu VM
+                item_status = properties.get(self._fact('qmpstatus'), item_status)
+        self.inventory.add_child(self._group('all_%s' % (item_status)), name)
```

Comment:
The brakets here have no effect. Either leave them away, or add a trailing comma to make this a proper tuple:
```suggestion
        self.inventory.add_child(self._group('all_%s' % (item_status, )), name)
```
(Same below.)